hooks/command

```diff
@@ -4,6 +4,11 @@ source $(dirname ${BASH_SOURCE})/common
 # Use default julia test running harness
 if [[ "${run_tests}" == "true" ]]; then
     echo "+++ :julia: Running tests"
+    if [[ -z "${package}" ]]; then
+        PKG="PackageSpec[]"
```

Comment:
why `PackageSpec[]` instead of the empty string?

Reply:
Interpolating an empty string would IIRC be equivalent to calling `Pkg.test("")` which means a package with an empty name.

Reply:
Sorry, I meant making `PKG` an empty string in bash (i.e. not `"\"\""`) so that it doesn't end up passing any argument in Julia.

```
$ export PKG=""
pkg/LLVM$ julia --project -e "using Pkg; Pkg.test($PKG);"
     Testing LLVM
```

Reply:
Okay, somehow I thought I tried that but apparently not.